source/data-formats/bson.txt

```diff
@@ -40,6 +37,27 @@ The code samples in this guide use the following BSON document as an example:
        "name" : "Mongo's Pizza"
    }
 
+BSON Data Types
+---------------
+
+BSON supports all JSON data structure types and adds support for types including 
+dates, different size integers, ObjectIds, and binary data. For a complete list of 
+supported types, see the :manual:`BSON Types </reference/bson-types>` page in the 
+Server Manual.
```

Comment:
I think we generally use this format (there might be a source constant already too)
```suggestion
MongoDB Server manual.
```